.github/mergify.yml

```diff
@@ -12,10 +14,26 @@ pull_request_rules:
         type: "APPROVE"
         message: "We've automatically approved this PR because the checks from the automated Pull Request have passed."
 
+  - name: "approve manual PRs that have passed checks"
+    conditions:
+      - "-check-failure~=.*alpine.*"
+      - "-check-failure~=.*package.*"
+      - "-check-failure~=.*matrix.*"
+      - "check-success~=.*build-and-push.*"
+      - "#files<=14"
```

Comment:
```suggestion
      - "#files<=14"
      - "files~=^vendor/[^/]+/VERSION"
```

Reply:
This rule just means that one of the changed files must be a `VERSION`. Not what we want.